holoviews/core/data/xarray.py

```diff
@@ -4,28 +4,34 @@
 from collections import OrderedDict
 
 import numpy as np
-import xarray as xr
-
-try:
-    import dask
-    import dask.array
-except ImportError:
-    dask = None
 
 from .. import util
 from ..dimension import Dimension, asdim, dimension_name
 from ..ndmapping import NdMapping, item_check, sorted_context
 from ..element import Element
 from .grid import GridInterface
 from .interface import Interface, DataError
+from .util import get_dask_array
+
 
 
 class XArrayInterface(GridInterface):
 
-    types = (xr.Dataset, xr.DataArray)
+    types = ()
 
     datatype = 'xarray'
 
+    @classmethod
+    def loaded(cls):
```

Comment:
This method makes sense but based on the way it is used, shouldn't it be called `available` instead of `loaded`? I thought the idea was that you can check `sys.modules` for availability and it is only loaded once actually imported...

Reply:
Slightly confused here, what is the difference between the semantics of ``available`` and ``loaded`` in your mind? Loaded currently checks if the dependency has been loaded, it does not check whether the library is importable (which is my interpretation of "available") because there is no way to check whether a library is importable without importing it.

Reply:
I understand it now, `loaded` is correct.